Cargo.toml

```diff
@@ -25,17 +26,18 @@ digest = { version = "0.10" }
 displaydoc = { version = "0.2.3", default-features = false }
 ethereum-types = { version = "0.14.1", features = ["impl-serde"] }
 generic-array = "0.14.7"
-jf-primitives = { git = "https://github.com/espressosystems/jellyfish" }
-jf-utils = { git = "https://github.com/espressosystems/jellyfish" }
+jf-relation = { git = "https://github.com/espressosystems/jellyfish", branch = "ecc-non-native-binyi" }
```

Comment:
Shall we just temporarily hold this PR until the work in jellyfish is merged into main? There's already a fork of jellyfish as the sequencer team has some [dependency issue](https://espresso.zulipchat.com/#narrow/stream/311645-Engineering/topic/.E2.9C.94.20Updating.20dependencies/near/364148033).

Reply:
Yes, let's first merge the PR in Jellyfish. I'll then change the dependency here.

Reply:
changed the dependency back in fde92dead922b9e9720c0946db977ecd77d08b9a